src/libraries/Native/Unix/configure.cmake

```diff
@@ -42,6 +42,12 @@ else ()
     message(FATAL_ERROR "Unknown platform. Cannot define PAL_UNIX_NAME, used by RuntimeInformation.")
 endif ()
 
+if(CLR_CMAKE_USE_SYSTEM_LIBUNWIND)
```

Comment:
source-build builds each component (coreclr, libraries, installer) separately and passes a separate set of args. It can skip passing `-DCLR_CMAKE_USE_SYSTEM_LIBUNWIND` to libraries. But this seems like a nice to have to make it possible to build all of runtime at once using the main `./build.sh` script in one call.

Reply:
It is kind of unfortunate that the main build script only supports single set of cmake args that it pushes to all three sub-parts (coreclr, installer, libraries). It would be nice to enhance it to enable specifying these flags separately for coreclr, installer and libraries so that we don't have to do tricks like this. But for this PR, it is fine as is.

Reply:
I filed https://github.com/dotnet/runtime/issues/42711 for tracking that